File: sections/samples/file_upload.js
const request = require('request');
const fs = require('fs');
const email = 'johndoe@email.com';
const password = 'secret';

var formData = {
    project_id: 123456,
    file: fs.createReadStream('/path/to/file.jpg')
};

request.post(
    {
        url: 'https://app.paymoapp.com/api/files',
        formData: formData,
        headers: {
            'Accept': 'application/json'
        },
        auth: {
            user: email,
            pass: password
        }
    },
    function (error, response, body) {
        if (!error) {
            console.log('New file URL: ' + JSON.parse(body).files[0].file);
        } else {
            console.log(error);
        }
    }
);

File: sections/samples/file_upload.py
import requests

email = 'johndoe@email.com'
password = 'secret'

payload = {'project_id': 123456}
headers = {'Accept': 'application/json'}
r = requests.post('https://app.paymoapp.com/api/files',
    data=payload,
    files={'file': open('/Users/ldim/Pictures/interiors/zooco-estudio-etoday-06.jpg', 'rb')},
    headers=headers,
    auth=(email, password))

print 'New File URL: %s' % r.json()['files'][0]['file']
